apps/admin/backend/src/tabulation/card_counts.test.ts

```diff
@@ -7,7 +7,10 @@ import {
   MockCastVoteRecordFile,
   addMockCvrFileToStore,
 } from '../../test/mock_cvr_file';
-import { tabulateScannedCardCounts } from './card_counts';
+import {
+  tabulateFullCardCounts,
+  tabulateScannedCardCounts,
+} from './card_counts';
 
 test('tabulateScannedCardCounts - grouping', () => {
```

Comment:
Might need to update test case description since we're testing through a different method now